### ISSUE TITLE
Bug: Non-breaking space shows up in node deletion dialog 

### ISSUE THREAD

**Issue by pKallert** [neos/neos-ui#3297]
### Description
Non-breaking spaces are shown in the title and the text of the node deletion dialog 

### Steps to Reproduce

1. Add a new text element, for example of type 'Neos.Demo:Content.Text'
2. Press enter a lot of times to create newlines 
3. Delete the element 

#### Expected behavior
The node deletion dialog shows up and asks if I want to delete the element 

#### Actual behavior
The node deletion dialog show up and a lot of &nbsp is shown: 
![Bildschirmfoto 2022-12-13 um 16 06 02](https://user-images.githubusercontent.com/91674611/207370984-8855ddf3-40a9-4713-8221-bce349ee482c.png)

**Comment by crydotsnake** [comment]
Is this a duplicate of #3295 ?

**Comment by pKallert** [comment]
No, this ist about the &nbsp sign showing up in the modal title and content.
The other one ist about the styling issue of content that overflows the dialog.

**Comment by grebaldi** [comment]
I was able to verify that this affects Neos UI 7.3:
![image](https://user-images.githubusercontent.com/2522299/209338050-60cfdb64-6fba-464b-a1cf-fff4458dc2b1.png)

**Comment by crydotsnake** [comment]
Could not reproduce this in Neos UI 8 and 7. When checkout the frontend, its not even rendered (not in the HTML).
